libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/model/eh/EHorizonMapper.kt

```diff
@@ -137,7 +137,8 @@ internal fun EdgeMetadata.mapToEHorizonEdgeMetadata(): EHorizonEdgeMetadata {
         countryCodeIso3,
         countryCodeIso2,
         stateCode,
-        isRightHandTraffic
+        isRightHandTraffic,
+        isOneway,
```

Comment:
Do we have tests for this mapper?

Reply:
no, I've cut a ticket https://github.com/mapbox/mapbox-navigation-android/issues/4297